admin-dev/themes/new-theme/js/components/grid/extension/position-extension.js

```diff
@@ -31,141 +31,145 @@ const $ = window.$;
+      });
     }
 
-    /**
-     * Add ID's to Grid table rows to make tableDnD.onDrop() function work.
-     *
-     * @private
-     */
-    _addIdsToGridTableRows() {
-        this.grid.getContainer()
-            .find('.js-grid-table .js-' + this.grid.getId() + '-position')
-            .each((index, positionWrapper) => {
-                const $positionWrapper = $(positionWrapper);
-                const rowId = $positionWrapper.data('id');
-                const position = $positionWrapper.data('position');
-                const id = `row_${rowId}_${position}`;
-                $positionWrapper.closest('tr').attr('id', id);
-            });
-    }
+    return positions;
+  }
 
-    /**
-     * Process rows positions update
-     *
-     * @param {String} url
-     * @param {Object} params
-     * @param {String} method
-     *
-     * @private
-     */
-    _updatePosition(url, params, method) {
-        const isGetOrPostMethod = ['GET', 'POST'].includes(method);
+  /**
+   * Add ID's to Grid table rows to make tableDnD.onDrop() function work.
+   *
+   * @private
+   */
+  _addIdsToGridTableRows() {
+    this.grid.getContainer()
+      .find('.js-grid-table .js-' + this.grid.getId() + '-position')
+      .each((index, positionWrapper) => {
+        const $positionWrapper = $(positionWrapper);
+        const rowId = $positionWrapper.data('id');
+        const position = $positionWrapper.data('position');
+        const id = `row_${rowId}_${position}`;
+        $positionWrapper.closest('tr').attr('id', id);
+        $positionWrapper.closest('td').addClass('js-drag-handle');
+      });
+  }
 
-        const $form = $('<form>', {
-            'action': url,
-            'method': isGetOrPostMethod ? method : 'POST',
-        }).appendTo('body');
+  /**
+   * Process rows positions update
+   *
+   * @param {String} url
+   * @param {Object} params
+   * @param {String} method
+   *
+   * @private
+   */
+  _updatePosition(url, params, method) {
+    const isGetOrPostMethod = ['GET', 'POST'].includes(method);
 
-        const positionsNb = params.positions.length;
-        let position;
-        for (let i = 0; i < positionsNb; ++i) {
-            position = params.positions[i];
-            $form.append(
-                $('<input>', {
-                    'type': 'hidden',
-                    'name': 'positions['+i+'][rowId]',
-                    'value': position.rowId
-                }),
-                $('<input>', {
-                    'type': 'hidden',
-                    'name': 'positions['+i+'][oldPosition]',
-                    'value': position.oldPosition
-                }),
-                $('<input>', {
-                    'type': 'hidden',
-                    'name': 'positions['+i+'][newPosition]',
-                    'value': position.newPosition
-                })
-            );
-        }
+    const $form = $('<form>', {
+      'action': url,
+      'method': isGetOrPostMethod ? method : 'POST',
```

Comment:
maybe overkill test.
```js
'method': method === 'GET' ? method : 'POST'
```

Reply:
yes, except it is also use downer to check if a `_method` input needs to be added